src/Components/Main/Main.jsx

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable no-unused-vars */
+import React, { useEffect } from "react";
+import Filters from "./Filters/Filters";
+import { useState } from "react";
+import Tasks from "../Tasks/Tasks";
+import styles from "./main.module.css";
+import AddModal from "./AddModal/AddModal";
+
+function Main({ searchValue }) {
+  const [showModal, setShowModal] = useState(false);
```

Comment:
`showModal` indicates this variable is a function that shows modal, but since it's a boolean flag it should be named as is
shouldShowModal or isModalOpen